src/services/SnapshotService.ts

```diff
@@ -81,4 +81,20 @@ export class SnapshotService {
       }
     })
   }
+
+  static async getSnapshotStatusAndSpace(spaceName?: string) {
+    spaceName = spaceName && spaceName.length > 0 ? spaceName : SNAPSHOT_SPACE
+    const values = await Promise.all([
```

Comment:
You can change this to `const [status, space] ...` to avoid reassigning later in lines 91/92.